src/OrchardCore.Modules/OrchardCore.Users/Views/Items/AssignUserRoleTask.Fields.Design.cshtml

```diff
@@ -0,0 +1,7 @@
+@using OrchardCore.Users.Workflows.Activities;
+@using OrchardCore.Workflows.ViewModels;
+@model ActivityViewModel<AssignUserRoleTask>
+<header>
+    <h4><i class="fa fa-user"></i>@Model.Activity.GetTitleOrDefault(() => T["Assign user to role"])</h4>
+</header>
+<em>@Model.Activity.UserName T["to role"] @Model.Activity.RoleName</em>
```

Comment:
```suggestion
<em>T["{0} to role {1}", Model.Activity.UserName, Model.Activity.RoleName]</em>
```

Because in RTL languages you might want to change the order